pkg/reconciler/autoscaling/hpa/resources/hpa.go

```diff
@@ -66,6 +71,19 @@ func MakeHPA(pa *v1alpha1.PodAutoscaler, config *autoscalerconfig.Config) *autos
 				},
 			}}
 		}
+
+	case autoscaling.Memory:
+		if target, ok := pa.Target(); ok {
+			memory := resource.MustParse(strconv.Itoa(int(math.Ceil(target))) + "Mi")
```

Comment:
Might be nicer to use resource.NewQuantity here since there's only one allowed format

Reply:
sounds good